app/resources/forms.js

```diff
@@ -367,3 +367,13 @@ function make_form_expandable(expandedSection) {
     return false;
   });
 }
+
+function form_confirmation() {
+   var personName = $('given_name').value + ' ' + $('family_name').value;
+   if (confirm('Do you want to create a record for ' + personName + '?\n' +
```

Comment:
Internationalize this message i.e., make the message ready to be translated into other languages.

You can do that by writing e.g.,
```js
  confirm('{% trans "Do you want to create a record for this person?:" %}');
```
It's actually tricky to translate a message with a placeholder like personName here in JavaScript. Here, I would recommend *escape* from the issue by breaking the line into two:

```
Do you want to create a record for this person?
  Hiroshi Ichikawa
```